mlflow/server/handlers.py

```diff
@@ -261,12 +265,67 @@ def initialize_backend_stores(backend_store_uri=None, default_artifact_root=None
     except UnsupportedModelRegistryStoreURIException:
         pass
 
+_STRING_TYPE_DEFAULT_SCHEMA = {"type": "string"}
+_INTEGER_TYPE_DEFAULT_SCHEMA = {"type": "integer"}
+_REQUIRED_PARAM_DEFAULT_SCHEMA = {"required": True}
```

Comment:
Adds some defaults so we can reuse these in a bunch of places

Reply:
Also: I'm not actually sure if it should be `True` or `"true"` here, but I think `True` works